### PR TITLE
Spec home page

### PR DESCRIPTION
**Pull request by PavanKumarN8**
Hi @sigadamvenkata, I have updated the "Home Page" Selector and added iOS, Android, and iPad methods in the same page.

Local Test Run iOS Results
image

Local Test Run Android Results
image

### CODE REVIEW

File: tests/feds/mobileHomePageSanity.test.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { test, beforeEach, afterEach, expect } from '@playwright/test';
```
Comment (Dli3) [inline tests/feds/mobileHomePageSanity.test.js:1]: I don't believe you need to import beforeEach and afterEach. You can use test.beforeEach and test.afterEach.

File: tests/feds/mobileHomePageSanity.test.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  beforeEach(async ({ page }) => {
+    home = new HomePageSanity(page);
+  });
+
+  afterEach(async ({ page }) => {
+    await page.close();
+  });
+  features.forEach((props) => {
+    test(`${props.name}, ${props.tags}, ${props.country}`, async ({ page, baseURL }) => {
+      console.info(`[FEDSInfo] Checking Page: ${props.country} = ${baseURL}${features[props.tcid].path}`);
```
Comment (Dli3) [inline tests/feds/mobileHomePageSanity.test.js:17]: Can't you just do props.path? 
`${baseURL}${props.path}`);`

File: tests/feds/mobileHomePageSanity.test.js (hunk 1 above)
Comment (PavanKumarN8) [reply tests/feds/mobileHomePageSanity.test.js]: Hi @Dli3, I have addressed the above comment and pushed the code.

File: tests/feds/mobileHomePageSanity.test.js (hunk 2 above)
Comment (PavanKumarN8) [reply tests/feds/mobileHomePageSanity.test.js]: Hi @Dli3, I have addressed the above comment and pushed the code.

**Review by JackySun9 — APPROVED** [review]

**Review by Dli3 — APPROVED** [review]

**Pull request merged**